modules/core/src/scenegraph/gltf-instantiator/gltf-instantiator.js

```diff
@@ -138,32 +164,45 @@ export default class GLTFInstantiator {
     const result = {};
 
     Object.keys(attributes).forEach(attrName => {
-      result[attrName] = this.createBuffer(attributes[attrName]);
+      result[attrName] = this.createAccessor(
+        attributes[attrName],
+        this.createBuffer(attributes[attrName].bufferView, this.gl.ARRAY_BUFFER)
+      );
     });
 
     if (indices) {
-      // TODO: use .target for target
-      result.indices = this.createBuffer(indices, {target: this.gl.ELEMENT_ARRAY_BUFFER});
+      result.indices = this.createAccessor(
+        indices,
+        this.createBuffer(indices.bufferView, this.gl.ELEMENT_ARRAY_BUFFER)
+      );
     }
 
     log.info(4, 'glTF Attributes', {attributes, indices, generated: result})();
 
-    return result;
+    return Object.assign({}, DEFAULT_ATTRIBUTES, result);
   }
 
-  createBuffer(accessor, extra = {}) {
-    // TODO: make sure we only create one buffer per buffer view
-    return new Buffer(
-      this.gl,
-      Object.assign({data: accessor.data, accessor: this.createAccessor(accessor)}, extra)
-    );
+  createBuffer(bufferView, target) {
+    const name = `_luma_buffer_${target}`;
```

Comment:
Make this a little bit more structured, by adding a named array instead of creating string keys?

`bufferView.lumaBuffers[target]?`